src/TabBar.js

```diff
@@ -177,7 +177,7 @@ export default class TabBar<T: *> extends React.Component<Props<T>, State> {
       return null;
     }
     return (
-      <Animated.Text style={[styles.tabLabel, this.props.labelStyle]}>
+      <Animated.Text style={[styles.tabLabel, this.props.labelStyle]} numberOfLines={this.props.numberOfLines}>
```

Comment:
A callback is provided to allow the dev to provide alternative label rendering, even if it's just copy & pasting the original code